rbx_reflector/Cargo.toml

```
@@ -0,0 +1,29 @@
+[package]
+name = "rbx_reflector"
+version = "0.1.0"
+edition = "2021"
+
```

We should probably add the remaining useful fields. I'm not sure if we plan to publish rbx_reflector but it's good practice.

```suggestion
description = "A program for generating a reflection database for rbx-dom's projects"
license = "MIT"
homepage = "https://github.com/rojo-rbx/rbx-dom"
repository = "https://github.com/rojo-rbx/rbx-dom.git"
authors = ["Nezuo"]
```

You may put whatever name you want in the authors field, this is just a suggestion. The format is `Name <email>` but the email is optional.